### DISCUSSION TITLE
How to set a ignore-pattern

### DISCUSSION THREAD

**Discussion by xyecoding** [remarkjs/remark#962] [Q&A] [answered]
I used `ignore-pattern=['-']` to ignore the format change of `-`. However, it does not work

**Comment by Murderlon** [comment, marked as answer]
If you want to change formatting of lists, this is what you want to change: [`remark-stringify#optionsbullet`](https://github.com/remarkjs/remark/tree/main/packages/remark-stringify#optionsbullet). If you are using `remark`, also checkout: [example-passing-options-to-remark-stringify](https://github.com/remarkjs/remark/tree/main/packages/remark#example-passing-options-to-remark-stringify).
